src/index.js

```diff
@@ -1,17 +1,26 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { ThemeProvider } from '@material-ui/core/styles';
 import App from './App';
+import theme from './theme';
+
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+	  <ThemeProvider theme={theme}>
+	    {/* CssBaseline kickstart an elegant, consistent, and simple baseline to build upon. */}
+	    <CssBaseline />
+	    <App />
+	  </ThemeProvider>
   </React.StrictMode>,
-  document.getElementById('root')
+  document.querySelector('#root'),
```

Comment:
What does the line do?

Reply:
Im not sure which specific line this comment refers to sorry - it's not obvious in the GH UI

My best guess is L19 but I'll wait for you to confirm!

Reply:
L11/19 these are two different ways to do the same thing.

Reply:
OK, so this was only changed because I cut'n'paste the M\aterialUI theme integration. I don't think it matters really.